### PR TITLE
Add delete operator example

### PR DESCRIPTION
**Pull request by isceruth**
Added examples of using delete operator with strict mode. 
Missing only cases with prototype chain ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#delete_and_the_prototype_chain)).

### CODE REVIEW

File: JavaScript/7-delete.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const org = 'HowProgrammingWorks';
+
+// console.log(delete org); // Raises SyntaxError in strict mode
```
Comment (tshemsedinov) [inline JavaScript/7-delete.js]: Why `console.log(delete org);` ? Just `delete org;`

File: JavaScript/7-delete.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const country = {
+  name: 'Chile',
+  ruler: 'Augusto Pinochet',
```
Comment (tshemsedinov) [inline JavaScript/7-delete.js]: :heart:

File: JavaScript/7-delete.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  born: 1915,
+  alive: false
+};
+
+delete country.ruler;
```
Comment (tshemsedinov) [inline JavaScript/7-delete.js]: ✯ :+1: ☭

File: JavaScript/7-delete.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+console.dir({ country });
+
+console.log(delete country.born);
+console.log(delete country.population);
```
Comment (tshemsedinov) [inline JavaScript/7-delete.js]: 💋

File: JavaScript/7-delete.js (hunk 1 above)
Comment (isceruth) [reply JavaScript/7-delete.js]: ¯\_(ツ)_/¯

**Comment by isceruth** [comment]
I left it to show that this operation returns true, no need in this?

**Comment by tshemsedinov** [comment]
By the way, @isceruth, "Minor improvements #2" isn't a good title for commit because #2 refers to issue "Arithmetic operators"

**Pull request merged**